osgp/platform/osgp-adapter-kafka-distributionautomation/src/main/resources/meterReading.avsc

```diff
@@ -0,0 +1,89 @@
+{
+  "type" : "record",
+  "name" : "MeterReading",
+  "namespace" : "org.opensmartgridplatform.adapter.kafka",
```

Comment:
I would expect distribution automation to be a part of the namespace

Reply:
Although this file will probably become obsolete, it might serve as an example for a new avsc. Therefor updated the namespace (added ".da.avro")